components/global_header/left_controls/product_menu/product_menu_list/product_menu_list.test.tsx

```diff
@@ -73,6 +76,29 @@ describe('components/global/product_switcher_menu', () => {
         expect(wrapper).toMatchSnapshot();
     });
 
+    test('should match snapshot with with cloud free', () => {
+        const props = {
+            ...defaultProps,
+            enableCustomUserGroups: true,
+            isCloud: true,
+            isCloudFreeEnabled: true,
+        };
+        const wrapper = shallow(<ProductMenuList {...props}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
+
+    test('should match snapshot with with cloud free trial', () => {
+        const props = {
+            ...defaultProps,
+            enableCustomUserGroups: true,
+            isCloud: true,
+            isCloudFreeEnabled: true,
+            isFreeTrial: true,
+        };
+        const wrapper = shallow(<ProductMenuList {...props}/>);
+        expect(wrapper).toMatchSnapshot();
+    });
```

Comment:
Could we assert on dom nodes or text content instead of a snapshot test? There's enough rendered content that someone verifying changes in the future is likely to skip reading over the 450 lines of snapshot that is added in favor of blindly doing a `npm test -- -u`, and the reviewers aren't likely to catch a mistaken snapshot update this large.